session.go

```diff
@@ -285,6 +283,35 @@ func (s *session) GetGlobalSysVar(ctx context.Context, name string) (string, err
 	return value, nil
 }
 
+// GetGlobalStatusVar implements RestrictedSQLExecutor.GetGlobalStatusVar interface.
+func (s *session) GetGlobalStatusVar(ctx context.Context, name string) (string, error) {
+	// TODO: get global status variables from store.
```

Comment:
You can use a table in system db to store status and init this table when doing bootstrap.